### PR TITLE
#15 Add Triple support

### PR DESCRIPTION
**Pull request by mjedynak**

### CODE REVIEW

File: kiwi-core/src/commonMain/kotlin/org/fromsource/kiwi/core/PairShould.kt [hunk 1]
```diff
@@ -1,8 +1,7 @@
 package org.fromsource.kiwi.core
 
 class PairShould<A, B>(val actual: Pair<A, B>) :
-        BeAny<Pair<A, B>, PairShould<A, B>>,
-        BeEqual<Pair<A, B>, PairShould<A, B>> {
```
Comment (mjedynak) [inline kiwi-core/src/commonMain/kotlin/org/fromsource/kiwi/core/PairShould.kt:5]: seems redundant

File: kiwi-json/src/commonTest/kotlin/org/fromsource/kiwi/json/ExtensionTest.kt [hunk 2]
```diff
@@ -11,7 +11,7 @@ class ExtensionTest {
                 .split('a', 0)
                 .should
                 .haveFirstEqualTo("This is a sentence".toCharList())
-                .haveSecondBeEqual("".toCharList())
+                .haveSecondEqualTo("".toCharList())
     }
```
Comment (mjedynak) [inline kiwi-json/src/commonTest/kotlin/org/fromsource/kiwi/json/ExtensionTest.kt:16]: should we add tests for Triple here as well?

File: kiwi-core/src/commonMain/kotlin/org/fromsource/kiwi/core/PairShould.kt (hunk 1 above)
Comment (torczuk) [reply kiwi-core/src/commonMain/kotlin/org/fromsource/kiwi/core/PairShould.kt:5]: 👍

File: kiwi-json/src/commonTest/kotlin/org/fromsource/kiwi/json/ExtensionTest.kt (hunk 2 above)
Comment (torczuk) [reply kiwi-json/src/commonTest/kotlin/org/fromsource/kiwi/json/ExtensionTest.kt:16]: No, this test is for checking `split` function used for json

**Review by torczuk — APPROVED** [review]

**Comment by torczuk** [comment]
Finally 👯

**Pull request merged**